### PR TITLE
Docs: add doc about storage class of object storage

### LINKED ISSUE #1043: Support s3 storageclass
**What would you like to be added**:
For s3-like object storage backend, storageclass is used to control redundancy level of a object (https://docs.aws.amazon.com/AmazonS3/latest/userguide/storage-class-intro.html) and it has been implemented by s3 client library. It would be great if we could have this property on juicefs format to let user have the choice of what level of redundancy they want and reduce the cost of s3.

**Why is this needed**:
I want to use “Reduced Redundancy” level of storageclass for non-important files.

### PR DESCRIPTION
**Pull request by xiaogaozi**
Fix #1043

### CODE REVIEW

**Review by yuhr123 — APPROVED** [review]

File: docs/en/reference/how_to_setup_object_storage.md [hunk 1]
```diff
@@ -57,6 +57,14 @@ JuiceFS also provides flexible support for this object storage service that dist
 
 Creating a file system using an internal Endpoint ensures better performance and lower latency, and for clients that cannot be accessed through an internal address, you can specify a public Endpoint to mount with the `--bucket` option.
 
+## Storage Class
+
+Object storage usually supports multiple storage classes, such as standard storage, infrequent access storage, and archive storage. When creating an object storage bucket you can choose the appropriate storage class according to your actual needs, or automatically convert the storage class of existing objects through lifecycle management. Regardless of the storage class used, it is supported as the underlying data storage for JuiceFS.
```
Comment (davies) [inline docs/en/reference/how_to_setup_object_storage.md:62]: We can't use the archive (or deep archive) storage class, the objects with that class is not accessible inmediately (requring a restore process).

Comment (xiaogaozi) [reply docs/en/reference/how_to_setup_object_storage.md:62]: I'll update the document later.

Comment (davies) [reply docs/en/reference/how_to_setup_object_storage.md:62]: Why not do that in this one?

Comment (xiaogaozi) [reply docs/en/reference/how_to_setup_object_storage.md:62]: Yes, I mean update this PR later.

**Pull request merged**